### PR TITLE
Bsampson/update seed data

### PR DESCRIPTION
**Pull request by besampson0**
### Description
Added more tasks to substitutions seed data. Also changed the name and ssn for appeals from sanitized json to be more obvious that this is not prod data.

### Acceptance Criteria
- [x] Code compiles correctly
- [x] Active and completed tasks have been added to substitution seed data.

### Testing Plan
1. Login in as cob admin or user.
2. search for appeal '54545454'
3. open a dispatched case and check that there are open and completed tasks.
4. open a pending case and check that there are open and completed tasks.

### CODE REVIEW

File: db/seeds/sanitized_json/appeal-126187.json [hunk 1]
```diff
@@ -38,12 +38,12 @@
       "id": 932150,
       "file_number": "000785014",
```
Comment (pbradin) [inline db/seeds/sanitized_json/appeal-126187.json:39]: Can't view this file number locally or in demo... it's erroring out on app/models/concerns/hearing_request_type_concern.rb:94.  So this isn't an issue with your change; it existed before.

**Comment by eileen-nava** [comment]
Hi!  Here’s what I see when I search for 54545454.

Screen Shot 2022-01-13 at 11 25 08 AM


For the dispatched cases, they showed only cancelled and completed tasks on the “Create Tasks page”.   I would suggest adding active tasks to the cases.

The “assigned to attorney” cases had a mix of active and closed tasks. 👍

File: db/seeds/sanitized_json/appeal-126187.json (hunk 1 above)
Comment (eileen-nava) [reply db/seeds/sanitized_json/appeal-126187.json:39]: I get the same error as Paula, and I agree that this error existed before the current PR.

Comment (besampson0) [reply db/seeds/sanitized_json/appeal-126187.json:39]: Fixed now! :)

Comment (pbradin) [reply db/seeds/sanitized_json/appeal-126187.json:39]: can see it now, thanks!

**Review by pbradin — APPROVED** [review]
Looks great!

**Pull request merged**